packages/client/src/objectSet/ObjectSetListenerWebsocket.test.ts

```diff
@@ -287,10 +287,13 @@ describe("ObjectSetListenerWebsocket", async () => {
                 "employeeLocation": GeotimeSeriesPropertyImpl {
                   "lastFetchedValue": {
                     "time": "111",
-                    "value": [
-                      100,
-                      200,
-                    ],
+                    "value": {
+                      "coordinates": [
+                        100,
+                        200,
+                      ],
+                      "type": "Point",
```

Comment:
This should catch this error next time if the shape ever changes accidentally

Reply:
Tests are the most valuable here because they tend to prevent regressions the best